app/code/Vladimirl/Chatter/view/frontend/web/css/source/_module.less

```diff
@@ -1,71 +1,78 @@
 & when (@media-common = true) {
```

Comment:
@Vladimir-L What's wrong with the code style again and again? Check Magento code style guides once more. There MUST be 4 (four) SPACES (not tabs). Not 2 spaces, but 4 spaces according to the Magento style guide and to how other Magento LESS files are built.